runbotics/runbotics-orchestrator-ui/src/src-landing/components/RichTextRenderer/RichTextRenderer.tsx

```diff
@@ -26,7 +30,16 @@ const RichTextRenderer: VFC<Props> = ({ content }) => {
             [BLOCKS.HEADING_4]: (_, children) => <Typography font="Roboto" className={styles.h4} variant="h4">{children}</Typography>,
             [BLOCKS.HEADING_5]: (_, children) => <Typography font="Roboto" className={styles.p} variant="h5">{children}</Typography>,
             [BLOCKS.HEADING_6]: (_, children) => <Typography font="Roboto" className={styles.p} variant="h6">{children}</Typography>,
-            [BLOCKS.PARAGRAPH]: (_, children) => <Typography font="Roboto" className={styles.p} variant="body3">{children}</Typography>,
+            [BLOCKS.PARAGRAPH]: (_, children) => {
+
+                if( children.toString().match(rex)){
```

Comment:
sometimes space is dangerous 

if( ch -> if(ch


![image](https://github.com/runbotics/runbotics/assets/69820069/b525c72c-7d39-4b7d-b2f8-d659ba384447)